### ISSUE TITLE
Ffmpeg Not Found on Latest openSUSE Tumbleweed

### ISSUE THREAD

**Issue by richardstevenhack** [mhogomchungu/media-downloader#342]
After a huge update yesterday of openSUSE Tumbleweed, Media Downloader (current version 4.0.0) can not find the installed ffmpeg. When I run Media Downloader with checks enabled, I get:

[media-downloader] Checking installed version of yt-dlp
[media-downloader] Found version: 2023.10.13
[media-downloader] Checking installed version of ffmpeg
[media-downloader] Failed to find version information, make sure "ffmpeg" is installed and works properly

Ffmpeg is installed in /usr/bin as shown:

rhack@localhost:/Data2/Work> whereis ffmpeg
ffmpeg: /usr/bin/ffmpeg /usr/share/ffmpeg /usr/share/man/man1/ffmpeg.1.gz

**Comment by mhogomchungu** [comment]
What is the output of the following command
```
ffmpeg -version
```

**Comment by richardstevenhack** [comment]
Well, this is weird:

rhack@localhost:/Data2/Work> ffmpeg -version
ffmpeg: error while loading shared libraries: libjack.so.0: cannot open shared object file: No such file or directory

rhack@localhost:/Data2/Work> /usr/bin/ffmpeg
/usr/bin/ffmpeg: error while loading shared libraries: libjack.so.0: cannot open shared object file: No such file or directory

rhack@localhost:/Data2/Work> whereis ffmpeg
ffmpeg: /usr/bin/ffmpeg /usr/share/ffmpeg /usr/share/man/man1/ffmpeg.1.gz

I have no idea what that means. Probably Tumbleweed screwing around again?

**Comment by mhogomchungu** [comment]
ffmpeg depends on libraries that are supposed to be available in your system and one of the library is "libjack.so.0" and this library is not available in your system.

One solution is to go to your distro's package manager and install "libjack". A best solution is to report ffmpeg as a broken package because it does not pull in off of its dependencies.

You can get a list of all libraries ffmpeg depends by running below command
```
ldd /usr/bin/ffmpeg
```

**Comment by richardstevenhack** [comment]
I did a search on RPMfind.net and found the package, installed it, and now Media Downloader finds ffmpeg.

Thanks, you've saved me again.

Unfortunately I don't know to report a broken package and right now I have a new PC build which has serious issues, so I'll have to rely on someone else to report that. In fact, it could be because one of the issues my PC is having is audio skipping possibly caused by USB problems on the ASUS motherboard and in the process of trying to debug that I messed around with the audio subsystems in openSUSE Tumbleweed, so it's possible I caused the problem. In other words, it will probably end up being my fault on this machine alone.

Anyway, thanks again.
